packages/cozy-client/src/manifest.spec.js

```diff
@@ -133,3 +188,262 @@ describe('sanitizeManifest', () => {
     })
   })
 })
+
+describe('getIdentifier', () => {
+  it('should return field having role=identifier', () => {
+    const fields = {
+      username: {
+        type: 'text'
+      },
+      id: {
+        type: 'text',
+        role: 'identifier'
+      }
+    }
+    expect(getIdentifier(fields)).toBe('id')
+  })
+
+  it('should return the first field', () => {
+    const fields = {
+      username: {
+        type: 'text'
+      },
+      id: {
+        type: 'text'
+      }
+    }
+
+    expect(getIdentifier(fields)).toBe('username')
+  })
+})
+
+describe('sanitizeFields', () => {
```

Comment:
sanitize ? 

You're not testing `sanitizeFields`. If you want to test it, you need to export it. This is my prefer-ed way instead of having to test the full `sanitize` to expect that `sanitizeFields` is working correctly.